tests/modules/test_gammacorrection.py

```diff
@@ -0,0 +1,19 @@
+import cellprofiler.modules.gammacorrection
+import numpy.testing
+import skimage.exposure
+
+instance = cellprofiler.modules.gammacorrection.GammaCorrection()
+
+
+def test_run(image, module, image_set, workspace):
+    module.x_name.value = "example"
```

Comment:
I'm seeing this a lot. It's not really used in the tests. Does it make more sense in `conftest.py`?